Cabal/Distribution/Package.hs

```diff
@@ -138,6 +141,39 @@ unPackageName = unUnqualComponentName . packageNameToUnqualComponentName
 mkPackageName :: String -> PackageName
 mkPackageName = PackageName . mkUnqualComponentName
 
+-- | A pkgconfig library name
```

Comment:
But what is a pkgconfig library name?! Answer: "a valid argument to the pkg-config utility".